code/game/objects/items/storage/marine_boxes.dm

```diff
@@ -1,47 +1,78 @@
-/obj/item/storage/box/heavy_armor
+/obj/item/storage/box/crate
+	name = "crate"
+	desc = "It's just an ordinary wooden crate."
+	icon = 'icons/obj/items/storage/storage.dmi'
+	icon_state = "case"
+	foldable = /obj/item/stack/sheet/wood
+
+// Copied from C:\Dev\SS13\TerraGov-Marine-Corps\code\game\objects\items\storage\storage.dm
+/obj/item/storage/box/crate/attack_self(mob/user)
+	//Clicking on itself will empty it, if it has the verb to do that.
+	if(allow_quick_empty)
+		quick_empty()
+		return
+
+	//Otherwise we'll try to fold it.
+	if ( length(contents) )
+		return
+
+	if ( !ispath(foldable) )
+		return
+
+	// Close any open UI windows first
+	for(var/mob/M in content_watchers)
+		close(M)
+
+	// Now make the plank
+	to_chat(user, span_notice("You broken [src] into a plank."))
+	new foldable(get_turf(src))
+	qdel(src)
+
+/obj/item/storage/box/crate/update_icon_state()
+	icon_state = initial(icon_state)
+	if(!length(contents))
+		icon_state = "empty_case"
```

Comment:
Write this as a ternary operator or something, there's no reason to be changing the icon state twice for no reason.

Reply:
Gotcha. So like if(!var/is_empty) kind of deal?
FYI I copied this from the line 377 to 380 from code/game/objects/items/storage/boxes.dm

Reply:
Actually should it be like this?
```
if(!length(contents))
		icon_state = "empty_case"
	else
		icon_state = initial(icon_state)
```

Reply:
that works as well